pages/admin/users.tsx

```diff
@@ -70,10 +70,15 @@ const AdminUsers: React.FC<QueryDataProps<AllUsersData>> = ({ queryData }) => {
           onClick={(value: string) => changeFilter(value, 'option')}
           currentOption={searchOption.option}
         >
-          Search By:
+          <label htmlFor="search">Search By:</label>
         </FilterButtons>
       </div>
-      <input type="text" className="form-control" onChange={handleChange} />
+      <input
+        type="text"
+        id="search"
```

Comment:
Why is the `id` added? Does it have something to do with the label being added?

Reply:
There are two ways to add label to html tag, wrap it around and use `for` tag (`htmlFor` in case of React). I can't simply wrap input with new label because it will change UI so I used the latter method. `Search by` is now label for `input` despite being separate non nested tags.

Reply:
Ah, thanks for the detailed explanation!